src/frontend/js/lib/util/mapper/formdataMapper.ts

```diff
@@ -1,4 +1,5 @@
 export const formdataMapper = <T>(data: FormData | T) => {
+    console.log("DATA: ", data);
```

Comment:
Is this debug line still needed?

Reply:
I just spotted that _literally_ as you put the comment in!